bucket/tinytask.json

```diff
@@ -17,7 +17,7 @@
     ],
     "pre_install": "if (!(Test-Path \"$persist_dir\\TinyTask.ini\")) { New-Item \"$dir\\TinyTask.ini\" -ItemType File | Out-Null }",
     "persist": "TinyTask.ini",
-    "checkver": "version ([\\d.]+)",
+    "checkver": "version.*?>([\\d.]+)</font>",
```

Comment:
Simply something like this should work without greediness or anything:

```suggestion
    "checkver": "TinyTask\\s+([\\d.]+)",
```

![image](https://user-images.githubusercontent.com/13260377/68287979-e91e1500-0083-11ea-83ff-8cb69b0499a0.png)